### PR TITLE
Add Auth0 integration overview to Auth0 tile

### PR DESCRIPTION
**Pull request by camicano**
### What does this PR do?

Adds more detail around the Datadog/Auth0 integration.

### Motivation

I wanted to highlight the power the integration to help drive more engagement.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: auth0/README.md [hunk 1]
```diff
@@ -2,7 +2,33 @@
...
+
+
+### Integration overview
+
+The integration with Datadog is enabled by Auth0’s Log Streaming. This capability works by sending batches of log events as they are generated in Auth0, giving customers up-to-date information about their Auth0 tenant. Log streaming is capable of delivering 10X more logs, and guarantees delivery with error handling. Today we are also releasing a generic webhook allowing you to deliver near real-time logs to most third-party tools.
+
+
+#### The integration with Datadog provides a number of important benefits:
+
+Visualize Auth0 Data Without Extra Development Time
```
Comment (ChristineTChen) [inline auth0/README.md:15]:
```suggestion
Visualize Auth0 Data Without Extra Development Time

```
Should this be in bold or a header? Needs new line to be on a separate line.

File: auth0/README.md [hunk 2]
```diff
@@ -2,7 +2,33 @@
...
+One of the key benefits of using Datadog is the ability to collect and visualize data in order to identify trends. Engineering teams use it to visualize error rates and traffic data. Security teams use it to visualize authorization traffic and set up alerts for high-risk actions.
+
+
+#### Identity data provides crucial insight to all of these use cases, allowing teams to better identify problems and make informed decisions.
+
+Make Informed Decisions About System Architecture and Development
```
Comment (ChristineTChen) [inline auth0/README.md:21]:
```suggestion
Make Informed Decisions About System Architecture and Development

```
Same for this line

File: auth0/README.md [hunk 3]
```diff
@@ -2,7 +2,33 @@
...
+The integration with Datadog is enabled by Auth0’s Log Streaming. This capability works by sending batches of log events as they are generated in Auth0, giving customers up-to-date information about their Auth0 tenant. Log streaming is capable of delivering 10X more logs, and guarantees delivery with error handling. Today we are also releasing a generic webhook allowing you to deliver near real-time logs to most third-party tools.
+
+
+#### The integration with Datadog provides a number of important benefits:
+
+Visualize Auth0 Data Without Extra Development Time
+One of the key benefits of using Datadog is the ability to collect and visualize data in order to identify trends. Engineering teams use it to visualize error rates and traffic data. Security teams use it to visualize authorization traffic and set up alerts for high-risk actions.
+
+
+#### Identity data provides crucial insight to all of these use cases, allowing teams to better identify problems and make informed decisions.
```
Comment (sarina-dd) [inline auth0/README.md:19]:
```suggestion
#### Identity data

Identity data provides crucial insight to all of these use cases, allowing teams to better identify problems and make informed decisions.
```

This is a pretty long header. Might be better to have a have a shortened header here and then have this be the lead-in sentence for this section.

File: auth0/README.md [hunk 4]
```diff
@@ -2,7 +2,33 @@
...
+
+Make Informed Decisions About System Architecture and Development
+By tracking identity trends over time, teams can make informed decisions about product development or system architecture. For example, using authentication data to determine which devices to prioritize development. Likewise, by tracking peak login times and geographies that users are accessing the app from, system architecture teams can determine when and where to scale up resources.
+
+
+####  Quickly Respond to Performance and Security Incidents
```
Comment (sarina-dd) [inline auth0/README.md:25]:
```suggestion
####  Quickly respond to performance and security incidents
```

**Comment by camicano** [comment]
@sarina-dd @ChristineTChen I have incorporated your changes. Would you mind reviewing again?

Thanks!

File: auth0/README.md [hunk 5]
```diff
@@ -2,7 +2,37 @@
 
 ## Overview
 
-Use the Datadog-Auth0 integration to view and analyze your log events from Auth0.
+Auth0, the identity platform for development teams, provides developers and enterprises with the building blocks they need to secure their applications.
+
+
+### Integration overview
```
Comment (ChristineTChen) [inline auth0/README.md:8]:
```suggestion
```

I think we can skip this section to allow the following  headers to be more prominent

File: auth0/README.md [hunk 6]
```diff
@@ -2,7 +2,37 @@
...
+
+The integration with Datadog is enabled by Auth0’s Log Streaming. This capability works by sending batches of log events as they are generated in Auth0, giving customers up-to-date information about their Auth0 tenant. Log streaming is capable of delivering 10X more logs, and guarantees delivery with error handling. You can now also release a generic webhook allowing you to deliver near real-time logs to most third-party tools.
+
+
+#### The integration with Datadog provides a number of important benefits:
```
Comment (ChristineTChen) [inline auth0/README.md:13]:
```suggestion
### The integration with Datadog provides a number of important benefits:
```

File: auth0/README.md [hunk 7]
```diff
@@ -2,7 +2,37 @@
...
+
+Visualize Auth0 Data Without Extra Development Time
```
Comment (ChristineTChen) [inline auth0/README.md:15]:
```suggestion
#### Visualize Auth0 Data Without Extra Development Time
```

File: auth0/README.md [hunk 8]
```diff
@@ -2,7 +2,37 @@
...
+
+One of the key benefits of using Datadog is the ability to collect and visualize data in order to identify trends. Engineering teams use it to visualize error rates and traffic data. Security teams use it to visualize authorization traffic and set up alerts for high-risk actions.
+
+
+#### Identity data
+
+Identity data provides crucial insight to all of these use cases, allowing teams to better identify problems and make informed decisions.
+
+Make Informed Decisions About System Architecture and Development
```
Comment (ChristineTChen) [inline auth0/README.md:24]:
```suggestion
#### Make Informed Decisions About System Architecture and Development
```

**Comment by camicano** [comment]
@ChristineTChen I have added your suggestions. Let me know if there is anything else :)

**Pull request merged**